src/reducers/authenticationReducer.js

```diff
@@ -82,16 +126,16 @@ export default (state = INITIAL_STATE, { type, payload }) => {
       );
     case SIGN_IN_SUCCEEDED: {
       return state
-        .setIn(['user'], { ...DEFAULT_USER, ...payload })
+        .setIn(['user'], fromJS({ ...DEFAULT_USER, ...payload }))
```

Comment:
Why the change to `fromJS`?

Reply:
To keep the object as immutable. Plus `fromJS` deeply transforms the properties to immutable objects.